.buildkite/pipeline.ml.yml

```diff
@@ -419,7 +419,7 @@
   commands:
     - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
     - ./ci/env/install-java.sh
-    - DATA_PROCESSING_TESTING=1 ARROW_VERSION=9.* ARROW_MONGO_VERSION=0.5.* ./ci/env/install-dependencies.sh
```

Comment:
Found that the current `pyarrowmongo` 0.5 is not compatible with Arrow 12, so need to upgrade to 1.0.x:

```
The conflict is caused by:
--
  | The user requested pyarrow==12.*
  | pymongoarrow 0.5.1 depends on pyarrow<9.1 and >=9
  | The user requested pyarrow==12.*
  | pymongoarrow 0.5.0 depends on pyarrow<9.1 and >=9
  | The user requested (constraint) pyarrow>=6.0.1
```